contracts/examples/AyiiProduct.sol

```diff
@@ -27,7 +27,7 @@ contract AyiiProduct is
 
     bytes32 public constant INSURER_ROLE = keccak256("INSURER");
 
-    uint256 public constant PERCENTAGE_MULTIPLIER = 2**24;
+    uint256 public constant PERCENTAGE_MULTIPLIER = 10**6;
```

Comment:
Is this a breaking change? This will have impact on all contracts stored on the blockchain as they now contain wrong values, correct?

Reply:
yes, you're correct this is a breaking change. unless we would migrate the existing data.
probably bad idea to do this during a season

Reply:
could be handled by defining that we do not merge develop into mine while the season is running
needs to be properly discussed

Reply:
Lets discuss it when you're back.